bugzilla_dashboard/component_queries.py

```diff
@@ -79,8 +81,29 @@ def build(out=""):
             params["include_fields"] = ["product", "component"]
             ComponentQuery(name, params).gather(results)
 
-        if out:
-            with open(out, "w") as Out:
+        if out_dir:
+            try:
+                os.mkdir(out_dir)
+            except OSError:
+                logger.error(f"Cannot create output directory: {out_dir}.")
+
+            with open(f"{out_dir}/{OUTPUT_FILE}", "w") as Out:
                 json.dump(results, Out)
 
         return results
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Retrieve data from Bugzilla for product::component"
+    )
+    parser.add_argument(
+        "-o",
+        "--output",
+        dest="out_dir",
+        action="store",
+        default="",
```

Comment:
To ease TC integration, you can specify a default as `os.environ.get('BZD_OUTPUT_PATH')`